s3same/__init__.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import travispy
+import yaml
+
+from .travis import travis_encrypt as _travis_encrypt
```

Comment:
I'm probably going to put back some variant on the exposed `travis_encrypt` function—I'd kind of like the pieces of `s3same`'s functionality that could be used in isolation to be available in the library form, even if they aren't exposed in the CLI tool—but if I weren't, I'm wondering if I should be doing something to indicate that imports are private/protected/internal in scope?